tests/integration/nm/ovs_test.py

```diff
@@ -53,7 +53,8 @@ def bridge_default_config():
 
```

Comment:
regarding the commit message: It mentions the host kernel - does the upgrade to 18.04 help here?

Reply:
No. Still got same errors. Seems need more time on debuging on this whole OVS stuffs.